scripts/boot-p2p.py

```diff
@@ -1,200 +1,187 @@
-        print(f"Failed to open or write to file {e}.")
-
-
 def boot_p2p_network():
-    try:
-        create_bootstrap_node()
-        time.sleep(20) # Give bootstrap node an early start
-        print("Starting peer nodes.")
-        create_peer_nodes()
-        return 0
-    except Exception as e:
-        print(e)
-        return 1
+    create_bootstrap_node()
+    time.sleep(30)  # Give bootstrap node an early start
+    print("Starting bonded nodes")
+    create_bonded_nodes()
+    print("Starting peer nodes.")
+    create_peer_nodes()
 
 
 def generate_validator_private_key():
-    ### Create --validator-private-key and --validator--public-key and add to bonds.txt # ed25519 eventually secp256k1 ###
-
-    # # pynacl for libsodium
-    # # import nacl # libsodium/ed25519 support
-    # from nacl.public import PrivateKey, PublicKey
-    # import nacl.encoding
-    # import nacl.signing
-    # private_key = PrivateKey.generate()
-    # encoded_private_key = private_key.encode(encoder=nacl.encoding.Base16Encoder).decode('utf-8').lower()
-    # encoded_public_key = private_key.public_key.encode(encoder=nacl.encoding.Base16Encoder).decode('utf-8').lower()
-    # signing_key = nacl.signing.SigningKey.generate() 
-    # verify_key = signing_key.verify_key
-    # encoded_private_key = signing_key.encode(encoder=nacl.encoding.Base16Encoder).lower()
-    # encoded_public_key = verify_key.encode(encoder=nacl.encoding.Base16Encoder).lower() 
-
-    # import ed25519
-    # signing_key, verifying_key = ed25519.create_keypair()
-    # encoded_private_key = signing_key.to_ascii(encoding="base16").decode('utf-8')
-    # encoded_public_key = verifying_key.to_ascii(encoding="base16").decode('utf-8')
-
-    # import ecdsa # secp256k1 suppport
-
-    # Using pre-generated validator key pairs by rnode. We do this because warning below  with python generated keys
-    # WARN  coop.rchain.casper.Validate$ - CASPER: Ignoring block 2cb8fcc56e... because block creator 3641880481... has 0 weight
-    f=open('scripts/demo-validator-private-public-key-pairs.txt')
-    lines=f.readlines()
-    line_number = random.randint(1,295)
-    encoded_private_key = lines[line_number].split()[0]
-    encoded_public_key = lines[line_number].split()[1]
-
-    # print(f"Populating bonds file {bonds_file}")
-    # bond_weight = random.randint(1,100)
-    # line = f"{encoded_public_key} {bond_weight}"
-    # print(line)
-    # try:
-    #     with open(bonds_file, 'a+') as f:
-    #         f.write(f"{line}\n")
-    # except IOError as e:
-    #     print(f"Failed to open or write to file {e}.") 
-
-    return encoded_private_key, encoded_public_key
-
-
-def populate_bonds_file():
-    print(f"Populating bonds file {bonds_file}")
-    try:
-        with open(bonds_file, 'a+') as f:
-            for line in bonds:
-                print(line)
-                f.write(f"{line}\n")
-    except IOError as e:
-        print(f"Failed to open or write to file {e}.") 
+    import ed25519
```

Comment:
Perhaps this can be moved to the top level?  Or is this function used only some of the times?